src/util.cpp

```diff
@@ -978,6 +973,14 @@ std::string ArgsManager::GetChainName() const
     return CBaseChainParams::MAIN;
 }
 
+void SetupChainParamsBaseOptions()
```

Comment:
Bike-shed: renamed to SetupChainParamsOptions, these are all chainparams options there are.

Reply:
Done 👍